From 70dca6e4cab4d631fd1b1ffc96e4293b94e3a5dc Mon Sep 17 00:00:00 2001
From: andrew <andrew@whartonlabs.com>
Date: Fri, 3 Feb 2017 11:26:16 +1100
Subject: [PATCH] Fixed TritonApi.resizeInstance jsdoc

Documented fields of the opts param we wrong (copy/paste error). Updated to correct field names and descriptions.
---
 lib/tritonapi.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/tritonapi.js b/lib/tritonapi.js
index a958064..848f5f7 100644
--- a/lib/tritonapi.js
+++ b/lib/tritonapi.js
@@ -2287,9 +2287,9 @@ TritonApi.prototype.deletePolicy = function deletePolicy(opts, cb) {
  *
  * @param {Object} opts
  *      - {String} id: Required. The instance name, short id, or id (a UUID).
- *      - {String} name: Required. The new instance name.
+ *      - {String} package: Required. The new package name, shortId or id (a UUID).
  *      - {Boolean} wait: Wait (via polling) until the rename is complete.
- *        Warning: A concurrent rename of the same instance can result in this
+ *        Warning: A concurrent resize of the same instance can result in this
  *        polling being unable to notice the change. Use `waitTimeout` to
  *        put an upper bound.
  *      - {Number} waitTimeout: The number of milliseconds after which to
-- 
2.21.0

